docs/charter.md

```diff
@@ -38,7 +78,7 @@ ICU4X will have an independent code base from ICU, and will operate independentl
 
 No!
```

Comment:
No! what?

Reply:
No to answer the question right above it.